### PR TITLE
[2.11.x] DDF-3625 Sanitized Logout Page URL Parameters

### PR DESCRIPTION
**Pull request by idperez**
#### What does this PR do?
Back-port https://github.com/codice/ddf/pull/2965
#### Who is reviewing it? 
@brjeter 
@bakejeyner 
@blen-desta 
#### Choose 2 committers to review/merge the PR. 
@andrewkfiedler
@coyotesqrl 
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: platform/security/servlet/security-servlet-logout/src/main/webapp/js/logout-response.js [hunk 1]
```diff
@@ -10,7 +10,7 @@
             var param = paramString.split('=');
 
             if (param[0] === "msg") {
-                $('#extramessage').html(decodeURIComponent(param[1].split('+').join(' ')));
+				$('#extramessage').text(decodeURIComponent(param[1].split('+').join(' '))).html();
```
Comment (bdeining) [inline platform/security/servlet/security-servlet-logout/src/main/webapp/js/logout-response.js]: Is this formatting correct?

**Comment by coyotesqrl** [comment]
build now

**Comment by coyotesqrl** [comment]
build now

(network failure during itests)

**Comment by LinkMJB** [comment]
Testing a potential fix for current network failures

build now

**Comment by coyotesqrl** [comment]
build now

(unrelated itest failures)

**Comment by coyotesqrl** [comment]
build now

(nexus network resolution problem)

**Comment by oconnormi** [comment]
Problems occurred when retrieving this pr from github (ghprb-plugin) 

Build Now

**Pull request merged**